lib/src/widgets/utils/ds_chip.widget.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSChip extends StatelessWidget {
```

Comment:
Will we not have documentation on this widget?

Reply:
it's no necessary, it's a simple widget